src/pages/Login.jsx

```diff
@@ -0,0 +1,44 @@
+import React from 'react'
+import { Link } from 'react-router-dom';
+import { FaAngleRight } from "react-icons/fa"
+
+export const Login = () => {
+    return (
+        <main>
+            <section className="login">
+                <form action="" method="get" className="form p-m m-m br-s">
+                    <h1 className="m-s p-s txt-center">Login</h1>
+
+                    <div className="flex-column">
+                        <label htmlFor="Email" className="m-xs" id='email1'>Email Address:</label>
+                        <input type="email" name="Email" className="input-box p-s m-xs br-s" placeholder="abc@xyz.com" id='email1' />
+                    </div>
```

Comment:
You can create a `Input` component

Reply:
will refactor that after completing p0 screens